RecoHGCal/TICL/plugins/PatternRecognitionbyCA.cc

```diff
@@ -184,6 +187,13 @@ void PatternRecognitionbyCA<TILES>::makeTracksters(
       tmp.setSeed(input.regions[0].collectionID, seedIndices[tracksterId]);
 
       std::copy(std::begin(effective_cluster_idx), std::end(effective_cluster_idx), std::back_inserter(tmp.vertices()));
+      // Propagate the correct graph connections
+      tmp.edges().reserve(ntuplet.size());
+      for (auto const &t : ntuplet) {
+        std::array<unsigned int, 2> edge = {
+            {(unsigned int)doublets[t].innerClusterId(), (unsigned int)doublets[t].outerClusterId()}};
```

Comment:
from 4.15 http://cms-sw.github.io/cms_coding_rules.html#4--technical-coding-rules-1
```suggestion
            {static_cast<unsigned int>(doublets[t].innerClusterId()), static_cast<unsigned int>(doublets[t].outerClusterId())}};
```